common/ballot.go

```diff
@@ -0,0 +1,12 @@
+package common
+
+type Ballot struct {
+	// ID of the voting decision.
+	ID []byte
+
+	// Number already voted inner ring nodes.
```

Comment:
`Public keys of already voted inner ring nodes` will be more accurate.

Reply:
Fixed